prboom2/src/r_bsp.c

```diff
@@ -732,6 +734,7 @@ static void R_Subsector(int num)
           if (frontsector->floorheight >= viewz)
             tmpsec = GetBestBleedSector(frontsector, BLEED_NONE);
           if (tmpsec == NULL &&
+              viewz > frontsector->floorheight &&
```

Comment:
Isn't this line redundant? If the difference is greater than the floor bleed threshold it must also be nonzero.

Reply:
Yeah, I was thinking in terms of unsigned math where you need to avoid overflow.